unison-src/transcripts/cycle-update-6.md

```diff
@@ -0,0 +1,28 @@
+Not yet working: properly updating implicit terms with conflicted names.
```

Comment:
AFAICT this is an alias, not a conflicted name; is that right?

Reply:
Whoops, ya. And it's being rejected for a bunk reason as you pointed out `(E)`. I'll make it a real conflicted term with a merge.